packages/replay/src/launchdarkly.ts

```diff
@@ -0,0 +1,123 @@
+import dbg from "./debug";
+import { initialize, LDClient, LDLogger } from "launchdarkly-node-client-sdk";
+
+const debug = dbg("replay:launchdarkly");
+
+type UserFeatureProfile = {
+  type: "user";
+  id: string;
+};
+
+type AnonymousFeatureProfile = {
+  type: "anonymous";
+  id: "anonymous";
+};
+
+type FeatureProfile = AnonymousFeatureProfile | UserFeatureProfile;
+
+class NoOpLogger implements LDLogger {
+  error() {}
+  warn() {}
+  info() {}
+  debug() {}
+}
+
+class LaunchDarkly {
+  private client: LDClient;
+  private initialized = true;
+  private lastCall = Date.now();
+
+  constructor() {
+    this.client = LaunchDarkly.initializeClient();
```

Comment:
One slight thing that still bothers me here is that this call want lead to setting up the reClose timer. It's only after the first call to `variant`/`identify` when we set it up. The CLI itself is shortlived and we are always meant to call either of those so it's not a big concern.

Reply:
Reintialziation/reclose logic is more relevant in the backend (where I copied this code from). Removing this whole thing.